keras_nlp/models/distilbert/distilbert_presets.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+backbone_presets = {
+    "distilbert_base_uncased_en": {
```

Comment:
is looks like we also have distilbert_base_cased_en

https://storage.googleapis.com/keras-nlp/models/distilbert_base_cased_en/model.h5
https://storage.googleapis.com/keras-nlp/models/distilbert_base_cased_en/vocab.txt

should we add that here too?

Reply:
Oh, I forgot about these. Adding it.